### PR TITLE
NO-JIRA: Additional logging for extract image and binaries

### PR DESCRIPTION
**Pull request by deepsm007**
/cc @openshift/test-platform

### CODE REVIEW

File: pkg/test/ginkgo/external.go [hunk 1]
```diff
@@ -123,9 +123,13 @@ func extractReleaseImageStream(logger *log.Logger, registryAuthFilePath string)
 		logger.Printf("Using target cluster release image %q", releaseImage)
 	}
 
+	extractStartTime := time.Now()
+	logger.Printf("Run image extract for release image %q at %v", releaseImage, extractStartTime)
 	if err := runImageExtract(releaseImage, "/release-manifests/image-references", tmpDir, registryAuthFilePath); err != nil {
 		return nil, fmt.Errorf("failed extracting image-references from %q: %w", releaseImage, err)
 	}
+	extractDuration := time.Since(extractStartTime)
+	logger.Printf("Run image extract completed for release image %q at %v", releaseImage, extractDuration)
```
Comment (stbenjam) [inline pkg/test/ginkgo/external.go:132]: I would put this in runImageExtract itself instead of surrounding the call to it

Comment (deepsm007) [reply pkg/test/ginkgo/external.go:132]: I'll let the current run finish and update the PR

File: pkg/test/ginkgo/external.go [hunk 2]
```diff
@@ -204,7 +204,9 @@ func extractBinaryFromReleaseImage(logger *log.Logger, releaseImageReferences *i
 }
 
 // runImageExtract extracts src from specified image to dst
-func runImageExtract(image, src, dst string, dockerConfigJsonPath string) error {
+func runImageExtract(image, src, dst string, dockerConfigJsonPath string, logger *log.Logger) error {
+	startTime := time.Now()
+	logger.Printf("Run image extract for release image %q at %v", image, startTime)
```
Comment (stbenjam) [inline pkg/test/ginkgo/external.go:209]: Can you include the "src" in the logging string too? That'll tell us what we're trying to extract

**Comment by stbenjam** [comment]
Thanks

/lgtm

**Pull request merged**